src/Microsoft.AspNet.Mvc.Core/Formatters/FormatterCollection.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.Mvc.Formatters
+{
+    public class FormatterCollection<TFormatter> : List<TFormatter>
```

Comment:
I'd really prefer that this inherit from `Collection<T>` like in the sample you provided. `List<T>` exposes extra surface area that we have to assume users take a dependency on.